src/H5Ppublic.h

```diff
@@ -361,6 +361,8 @@ H5_DLL herr_t      H5Pget_mdc_log_options(hid_t plist_id, hbool_t *is_enabled, c
                                           size_t *location_size, hbool_t *start_on_access);
 H5_DLL herr_t      H5Pset_evict_on_close(hid_t fapl_id, hbool_t evict_on_close);
 H5_DLL herr_t      H5Pget_evict_on_close(hid_t fapl_id, hbool_t *evict_on_close);
+H5_DLL herr_t      H5Pset_file_locking(hid_t fapl_id, hbool_t use_file_locking, hbool_t ignore_when_disabled);
+H5_DLL herr_t H5Pget_file_locking(hid_t fapl_id, hbool_t *use_file_locking, hbool_t *ignore_when_disabled);
```

Comment:
Spacing seems odd here.  Not critical, but curious as to why.

Reply:
I didn't re-run the formatter since I was moving code from formatted source to formatted source.